bundles/org.pitest.pitclipse.runner/src/org/pitest/pitclipse/runner/results/summary/SummaryResultListener.java

```diff
@@ -59,12 +58,20 @@ public void runStart() {
 
     @Override
     public void handleMutationResult(ClassMutationResults results) {
-        List<ClassName> classUnderTest = Collections.singletonList(results.getMutatedClass());
-        int coveredLines = coverage.getNumberOfCoveredLines(classUnderTest);
-        for (ClassInfo info : coverage.getClassInfo(classUnderTest)) {
-            ClassSummary classSummary = ClassSummary.from(results, info, coveredLines);
-            result = result.update(classSummary);
-        }
+        List<ClassName> classesUnderTest = Collections.singletonList(results.getMutatedClass());
+        
+//        long totalNumberOfCoveredLines = classesUnderTest.stream().map(coverage::getCoveredLines).flatMap(Collection::stream).count();
```

Comment:
shall we remove these old commented lines? Or keep them for our reference?

Reply:
I'll remove them during cleanup